### DISCUSSION TITLE
UT 설문 공지

### DISCUSSION THREAD

**Discussion by Kyara0324** [nbc-final-a10/TripBuddy#211] [자유게시판]
안녕하세요, React 5기 "Trip Buddies" 웹 사이트를 제작 중인 여행가고싶조입니다.

저희가 약 3주간 개발한 사이트에 대한 사용자 테스트 설문을 진행하고 있습니다.

설문에 성실히 답변해주신 분들을 추첨하여 다음과 같은 상품을 드립니다.

- 1등 - [ 배민 5만원권 ]
- 2등 - [ 배민 3만원권 ]
- 3등 - [ 배민 2만원권 ]
- 그 외 16분께 - [ 스타벅스 카페라떼 기프티콘 ]

--
- 현재는 모바일 버전으로 개발되어 핸드폰에서 테스트하는 것을 추천드립니다!
- 또는 PC 화면에서 F12를 눌러 개발자모드의 모바일 환경으로 테스트도 가능합니다!

[ Trip Buddies 웹 서비스 간략 소개 ]
여행족들을 위한 여행 친구 파티 매칭 서비스입니다.
혼자 여행하기는 쓸쓸하지만, 주변에 여행을 같이 갈 만한 사람이 없을 땐?
Trip Buddies에서 여행 친구를 모집해 함께 여행을 떠나보세요!

- 설문 대상: 모두
- 설문 기간: 8월 7일(수) ~ 8월 11일(일) 23:59 까지
- 소요 시간: 약 5~10분
- 사이트 링크: https://nbc-final.vercel.app/
- 설문 링크: https://forms.gle/guH9pPTju6qvZYw99

많은 참여 부탁드립니다!

**Comment by Captain-Kim** [comment]
![tripbuddy_UT-notice-image](https://github.com/user-attachments/assets/d5164642-7e1a-481c-8e89-e5f3c6e44486)

**Comment by Captain-Kim** [comment]
안녕하세요, React 5기 **"TripBuddies"** 웹 사이트를 제작 중인 여행가고싶조입니다.

저희가 약 3주간 개발한 사이트에 대한 사용자 테스트 설문을 진행하고 있습니다.

설문에 성실히 답변해주신 분들을 추첨하여 다음과 같은 상품을 드립니다.

🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥

🥇 1등 - **[ 배민 5만원권 ]**
🥈 2등 - **[ 배민 3만원권 ]**
🥉 3등 - **[ 배민 2만원권 ]**
그 외 16분께 - **[ 스타벅스 카페라떼 기프티콘 ]**

🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥🔥

현재는 모바일 버전으로 개발되어 **모바일에서 테스트하는 것을 추천**드립니다!
또는 PC 화면에서 F12를 눌러 개발자모드의 모바일 환경으로 테스트도 가능합니다!

💙 **[ Trip Buddies 웹 서비스 간략 소개 ]** 💙

여행족들을 위한 여행 친구 파티 매칭 서비스입니다.
혼자 여행하기는 쓸쓸하지만, 주변에 여행을 같이 갈 만한 사람이 없을 땐?
Trip Buddies에서 여행 친구를 모집해 함께 여행을 떠나보세요!

- **<설문 대상>** : 누구나
- **<설문 기간>** : 8월 7일(수) ~ 8월 11일(일) 23:59 까지
- **<소요 시간>** : 약 5~10분
- **<사이트 링크>** : https://nbc-final.vercel.app/
- **<설문 링크>** : https://forms.gle/guH9pPTju6qvZYw99

많은 참여 부탁드립니다!

_[장원영의 축복] 이 글에 스티커 붙여주시면 개당 취업률 1%씩 버프 받습니다._

**[기타 문의]** bdohhhhh@gmail.com
